tsl/src/compression/create.h

```diff
@@ -15,10 +15,12 @@
 #define COMPRESSION_COLUMN_METADATA_COUNT_NAME COMPRESSION_COLUMN_METADATA_PREFIX "count"
 #define COMPRESSION_COLUMN_METADATA_SEQUENCE_NUM_NAME                                              \
 	COMPRESSION_COLUMN_METADATA_PREFIX "sequence_num"
-#define COMPRESSION_COLUMN_METADATA_MIN_COLUMN_NAME "min"
-#define COMPRESSION_COLUMN_METADATA_MAX_COLUMN_NAME "max"
+
 #define COMPRESSION_COLUMN_METADATA_PATTERN_V1 "_ts_meta_%s_%d"
 
+/* Version 2 is indexed by the uncompressed chunk attno, not by the orderby index. */
+#define COMPRESSION_COLUMN_METADATA_PATTERN_V2 "_ts_meta_v2_%s_%d"
```

Comment:
Should we use the column name here instead of thte attno?

Reply:
dump/restore may change attno so i dont think its safe to use in this context

Reply:
Changed to use the column names.